src/GraphBLAS-sharp.Backend/Objects/Matrix.fs

```diff
@@ -7,30 +7,30 @@ type MatrixFormat =
     | COO
     | CSC
 
-type Matrix<'a when 'a: struct> =
-    | MatrixCSR of CSRMatrix<'a>
-    | MatrixCOO of COOMatrix<'a>
-    | MatrixCSC of CSCMatrix<'a>
+type ClMatrix<'a when 'a: struct> =
```

Comment:
What is the reason to add `Cl` prefix?

Reply:
If this prefix is necessary, it may be better to do something like this:
```
[<RequireQualifiedAccess>]
type ClMatrix<'a when 'a: struct> =
    | CSR of CSRMatrix<'a>
    | COO of COOMatrix<'a>
    | CSC of CSCMatrix<'a>
```